sdk/metric/processor/basic/basic.go

```diff
@@ -113,7 +113,7 @@ type (
 
 var _ export.Processor = &Processor{}
 var _ export.Checkpointer = &Processor{}
-var _ export.CheckpointSet = &state{}
+var _ export.Reader = &state{}
```

Comment:
At line 93 (not shown here), comment still refers to `state.RWMutex` implementing locking for the `CheckpointSet` interface. I believe this should be changed to `Reader` interface.

Reply:
I've replaced CheckpointSet with Reader everywhere now.